src/main/java/edu/cornell/kfs/fp/service/impl/RecurringDisbursementVoucherDocumentServiceImpl.java

```diff
@@ -409,8 +417,137 @@ public Object call() throws WorkflowException {
             throw new RuntimeException("cancelDVAsSystemUser() Unable to cancel DV: " + dvDocumentNumber, e);
         }
     }
+
+    @Override
+    public boolean autoApproveDisbursementVouchersSpawnedByRecurringDvs() {
+        LOG.info("autoApproveDisbursementVouchersSpawnedByRecurringDvs: Entered.");
+        int approvalCount = 0;
+        int errorCount = 0;
+        List <String> dvDocIdCausingError = new ArrayList<String>();
```

Comment:
probably would make more sense to name this dvDocIdsCausingError since it might hold more than one id?